java/daikon/Daikon.java

```diff
@@ -149,12 +149,10 @@
 import daikon.inv.unary.sequence.SeqIndexIntLessEqual;
 import daikon.inv.unary.sequence.SeqIndexIntLessThan;
 import daikon.inv.unary.sequence.SeqIndexIntNonEqual;
-import daikon.inv.unary.string.CompleteOneOfString;
-import daikon.inv.unary.string.OneOfString;
-import daikon.inv.unary.string.PrintableString;
-import daikon.inv.unary.stringsequence.CommonStringSequence;
-import daikon.inv.unary.stringsequence.EltOneOfString;
-import daikon.inv.unary.stringsequence.OneOfStringSequence;
+import daikon.inv.unary.string.*;
```

Comment:
Please don't introduce wildcard imports.  The Daikon style is to not use them.

Reply:
Fixed in Commit https://github.com/codespecs/daikon/pull/497/commits/a1663546c73ae48608658b86a9dfbd916736698e

Reply:
Thanks.  If you address a comment, you can just click the "Resolve conversation" button.  Let's reserve replies for conveying more information.